Dockerfile.test

```diff
@@ -3,6 +3,9 @@ FROM ubuntu:18.04
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    software-properties-common \
+    && add-apt-repository ppa:git-core/ppa -y \
```

Comment:
I had to update the `git` version in the test container, it was 2.17, needs to be >= 2.25